pkg/minikube/cluster/cluster_provisioner.go

```diff
@@ -0,0 +1,293 @@
+/*
+Copyright (C) 2016 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This provisioner only for RHEL/CentOS iso which created to consume
+// with minishift
+
+package cluster
```

Comment:
@hferentschik Yes, I was going to ask same question, will make it part of different pkg.